gdal/ogr/ogrsf_frmts/mssqlspatial/ogr_mssqlspatial.h

```diff
@@ -378,6 +378,10 @@ class OGRMSSQLSpatialTableLayer final: public OGRMSSQLSpatialLayer
     int                 FetchSRSId();
 
     void                SetUseCopy(int bcpSize) { bUseCopy = TRUE; nBCPSize = bcpSize; }
+
```

Comment:
can you insert before the declarations of StartCopy() and EndCopy() (before each line) a comment line
// cppcheck-suppress functionStatic
so that cppcheck doesn't complain ?
See ogr/ogrsf_frmts/ogrsf_frmts.h for an example